test/cts/from_nnapi/README.md

```diff
@@ -0,0 +1,99 @@
+
+## Test Generator Tool
+This [test_generator](./test_generator) tool is for converting existed native 
+[NNAPI CTS](https://android.googlesource.com/platform/frameworks/ml/+/refs/tags/android-cts-10.0_r5/nn/runtime/test/specs/) of V1_0, V1_1 and V1_2 versions to these tests for WebNN API.
+
+### NNAPI Operations being map to WebNN API Operations Tables
+* Part I 
+
+The `test_generator` tool has supported to convert those tests of following
+NNAPI operations to the tests for such WebNN API operations of first wave.
+
+| NNAPI                         | WebNN API (first wave ops)    |
+|:------------------------------|:------------------------------|
+| RELU1                         | clamp                         |
+| RELU6                         | clamp                         |
+| CONCATENATION                 | concat                        |
+| ADD                           | add [+ relu/clamp]            |
+| SUB                           | sub [+ relu/clamp]            |
+| MUL                           | mul [+ relu/clamp]            |
+| DIV                           | div [+ relu/clamp]            |
+| MAXIMUM                       | max                           |
+| MINIMUM                       | min                           |
+| EXP                           | exp                           |
+| LOGISTIC                      | sigmoid                       |
+| SQRT                          | sqrt                          |
+| TANH                          | tanh                          |
+| AVERAGE_POOL_2D               | averagePool2d [+ relu/clamp]  |
+| MAX_POOL_2D                   | maxPool2d  [+ relu/clamp]     |
+| CONV_2D                       | conv2d [+ add + relu/clamp]   |
+| DEPTHWISE_CONV_2D             | conv2d [+ add + relu/clamp]   |
+| GROUPED_CONV_2D               | conv2d [+ add + relu/clamp]   |
+| RELU                          | relu                          |
+| RESHAPE                       | reshape                       |
+| SLICE                         | slice                         |
+| SOFTMAX                       | softmax + exp                 |
+| SPLIT                         | split                         |
+| SQUEEZE                       | squeeze                       |
+| TRANSPOSE                     | transpose                     |
+
+* Part II
+
+And there're these following NNAPI operations which could be map to others
+WebNN API operations.
+
+| NNAPI                         | WebNN API                     |
+|:------------------------------|:------------------------------|
+| ABS                           | abs                           |
+| FLOOR                         | floor                         |
+| LOG                           | log                           |
+| NEG                           | neg                           |
+| SIN                           | sin                           |
+| L2_POOL_2D                    | l2Pool2d  [+ relu/clamp]      |
```

Comment:
What's the difference between Part I and Part II?

Reply:
@huningxin Those ops of Part I are the first wave ops, and ops of Part II will be next waves ops. I've updated the README with this description, thanks.